src/universal/modules/meeting/components/MeetingAgendaCards/MeetingAgendaCards.js

```diff
@@ -9,33 +9,22 @@ import {ACTIVE} from 'universal/utils/constants';
 import withHotkey from 'react-hotkey-hoc';
 import OutcomeOrNullCard from 'universal/components/OutcomeOrNullCard/OutcomeOrNullCard';
 
-const handleAddActionFactory = (teamMemberId, agendaId) => () => {
-  const [, teamId] = teamMemberId.split('::');
-  const newAction = {
-    id: `${teamId}::${shortid.generate()}`,
-    teamMemberId,
-    sortOrder: 0,
-    agendaId
-  };
-  cashay.mutate('createAction', {variables: {newAction}});
-};
-
 const handleAddProjectFactory = (teamMemberId, agendaId) => () => {
   const [, teamId] = teamMemberId.split('::');
   const newProject = {
     id: `${teamId}::${shortid.generate()}`,
     status: ACTIVE,
     teamMemberId,
     sortOrder: 0,
-    agendaId
+    agendaId,
```

Comment:
hmmm, lemme track down that lint rule, we should have those be automatically fixed